### ISSUE TITLE
2023 Prius Prime - Running the Remote Start service has broken remote start for Toyota app - BEWARE!!

### ISSUE THREAD

**Issue by caldorf** [widewing/ha-toyota-na#61]
As per the title, there is no option to remote start my 2023 Prius Prime. I can through the toyota app no problem, I"m using it regualraly to warm up my vehicle in the mornings but I would like to be able to trigger it via home assistant rather then Toyota's clunky app. Everything else seems to be working well, I can unlock/lock just finr, but no remote start.

**Comment by widewing** [comment]
It's not exposed as a button, you can find it in the service in dev tools

**Comment by caldorf** [comment]
Ah got it, the issue is when I call that service all I get is a notification on my phone from the Toyota app saying: "There was an error while processing your request, please try again, if you receive this error repeatedly then please contact your dealer" and the car does not remote start. I don't think its an authentication issue as I can still unlock and lock the car through HA

**Comment by caldorf** [comment]
UPDATE: After running the call service and it failing it has entirely broken my remote start. I can't remote start at all now, not even in the Toyota App. I have called Toyota support and they can't fix it either, they said I need to take it in to the dealership to have the entire car reset...

**Comment by jgdf42** [comment]
should i be worried about it this? i was going to start using the remote start function in this integration but now im afraid.

**Comment by caldorf** [comment]
Toyota ended up needing to fully reset my car as well as my app. It took over a week for me to be able to use the app again as well as a full trip to my dealership. It was bad

Not only was setting up the integration a pain but it was by far the most unreliable integration I have ever used in Home Assistant, constantly not reporting the correct information, going offline, breaking and not working. It broke my Toyota app and constantly got me temporarily banned from using the app for pulling too much data from the API

To be clear I don't blame the dev for this, I blame Toyota who has made one of the worse apps for their cars I have ever seen.

Basically, if you do want to do this just know that it will give you bad and broken data so I don't think its worth it imo

**Comment by rickestrada** [comment]
Reviving this thread. Just got into a 2024 Rav4, anyone have similar issue with remote start becoming unusable even in the Toyota app after calling service from HA?

**Comment by Kelinni** [comment]
I use this fully and successfully with lock, unlock, sensors, and remote start on my 2024 Prius Limited.

Sorry for your trouble. Hope you find a solution.

On Jul 7, 2024, at 6:42 PM, Rick Estrada ***@***.***> wrote:

﻿

Reviving this thread. Just got into a 2024 Rav4, anyone have similar issue with remote start becoming unusable even in the Toyota app after calling service from HA?

**Comment by dkmcgowan** [comment]
I have used this feature as well and it works and can't see how calling the API in the way it's called here could cause any issue to the vehicle, while not impossible, I suspect it was some other issue in the system in the car that a reset resolved.  To even have a public API from them that could cause that kind of damage would have been a bug on their part.
